### PR TITLE
Package cmarker.0.3.0

### PR DESCRIPTION
**Pull request by pablo-meier**
### `cmarker.0.3.0`
Bindings for a local installation of CMark

---
* Homepage: https://github.com/pablo-meier/ocaml-cmark
* Source repo: git+https://github.com/pablo-meier/ocaml-cmark.git
* Bug tracker: https://github.com/pablo-meier/ocaml-cmark/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

File: packages/cmarker/cmarker.0.3.0/opam [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+maintainer: "Pablo Meier <pablo.a.meier@gmail.com>"
+authors: "Jonathan Chan <jyc@fastmail.fm>"
+homepage: "https://github.com/pablo-meier/ocaml-cmark"
+synopsis: "Bindings for a local installation of CMark"
+bug-reports: "https://github.com/pablo-meier/ocaml-cmark/issues"
+license: "BSD 2-Clause"
+dev-repo: "git+https://github.com/pablo-meier/ocaml-cmark.git"
+build: [ "dune" "build" "-p" name]
+depends: [
+  "ctypes" {>= "0.4.1"}
```
Comment (mseri) [inline packages/cmarker/cmarker.0.3.0/opam]:
```suggestion
  "ocaml"
  "dune"
  "ctypes" {>= "0.4.1"}
```

File: packages/cmarker/cmarker.0.3.0/opam [hunk 2]
```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+maintainer: "Pablo Meier <pablo.a.meier@gmail.com>"
+authors: "Jonathan Chan <jyc@fastmail.fm>"
+homepage: "https://github.com/pablo-meier/ocaml-cmark"
+synopsis: "Bindings for a local installation of CMark"
+bug-reports: "https://github.com/pablo-meier/ocaml-cmark/issues"
+license: "BSD 2-Clause"
+dev-repo: "git+https://github.com/pablo-meier/ocaml-cmark.git"
+build: [ "dune" "build" "-p" name]
```
Comment (kit-ty-kate) [inline packages/cmarker/cmarker.0.3.0/opam]:
```suggestion
build: [ "dune" "build" "-p" name "-j" jobs]
```
Users should be able to ask for a specific number of core being used.

File: packages/cmarker/cmarker.0.3.0/opam [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+depends: [
+  "ctypes" {>= "0.4.1"}
+  "ctypes-foreign" {>= "0.4.0"}
+  "utop" {= "2.4.0"}
+  "ocamlformat" {= "0.11.0"}
```
Comment (kit-ty-kate) [inline packages/cmarker/cmarker.0.3.0/opam]:
```suggestion
```
How is that required? I don't see any use of this package

File: packages/cmarker/cmarker.0.3.0/opam [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+homepage: "https://github.com/pablo-meier/ocaml-cmark"
+synopsis: "Bindings for a local installation of CMark"
+bug-reports: "https://github.com/pablo-meier/ocaml-cmark/issues"
+license: "BSD 2-Clause"
+dev-repo: "git+https://github.com/pablo-meier/ocaml-cmark.git"
+build: [ "dune" "build" "-p" name]
+depends: [
+  "ctypes" {>= "0.4.1"}
+  "ctypes-foreign" {>= "0.4.0"}
+  "utop" {= "2.4.0"}
```
Comment (kit-ty-kate) [inline packages/cmarker/cmarker.0.3.0/opam]: Could you untighten the constraints a little. This seems a little tight for one use of `UTop_main.main`

File: packages/cmarker/cmarker.0.3.0/opam (hunk 3 above)
Comment (pablo-meier) [reply packages/cmarker/cmarker.0.3.0/opam]: I was using it as a lint step; this was to have it installed in my local switch. I can remove it from here 🙂

**Comment by pablo-meier** [comment]
@kit-ty-kate @mseri Incorporated the suggestions, thanks for them, and your patience! 😄

File: packages/cmarker/cmarker.0.3.0/opam [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+authors: "Jonathan Chan <jyc@fastmail.fm>"
+homepage: "https://github.com/pablo-meier/ocaml-cmark"
+synopsis: "Bindings for a local installation of CMark"
+bug-reports: "https://github.com/pablo-meier/ocaml-cmark/issues"
+license: "BSD 2-Clause"
+dev-repo: "git+https://github.com/pablo-meier/ocaml-cmark.git"
+build: [ "dune" "build" "-p" name "-j" jobs]
+depends: [
+  "ocaml"
+  "dune"
```
Comment (kit-ty-kate) [inline packages/cmarker/cmarker.0.3.0/opam]:
```suggestion
  "dune" {>= "1.11"}
```
Your dune-project file says you require at least dune 1.11

**Comment by pablo-meier** [comment]
Hi!

Travis seems to be [failing one of its checks](https://travis-ci.org/ocaml/opam-repository/jobs/575086646), which I can try to take a crack at debugging sometime:

```
+sudo apt-get install --no-install-recommends -y bubblewrap
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package bubblewrap
```

otherwise, let me know if there's other action you'd like me to take 😄

File: packages/cmarker/cmarker.0.3.0/opam (hunk 4 above)
Comment (kit-ty-kate) [reply packages/cmarker/cmarker.0.3.0/opam]: any thoughts on that?

**Comment by kit-ty-kate** [comment]
I've untighten the constraints on utop. `UTop_main.main` has been available since at least the oldest version of utop installable in opam (`utop.1.2.1`)

**Comment by kit-ty-kate** [comment]
Could you return the fix in the opam file upstream?

**Comment by kit-ty-kate** [comment]
Oh I didn't think you did it upstream already. Thanks

**Pull request merged**